src/main/java/com/conveyal/datatools/manager/models/Project.java

```diff
@@ -72,6 +72,11 @@
      */
     public Set<AutoDeployType> autoDeployTypes = new HashSet<>();
 
+    /**
+     * Whether to auto-deploy feeds that have critical errors.
+     */
+    public boolean autoDeployWithCriticalErrors = false;
```

Comment:
Is this a field that we are going to add into the UI or is it just for testing?

Reply:
Yes, see https://github.com/ibi-group/datatools-ui/pull/637